responsive.css

```diff
@@ -0,0 +1,191 @@
+html, body {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+    color: rgb(75, 75, 75);
+}
+
+.navPane{
+    width: 100vw;
+    height: 30px;
+    position: fixed;
+    margin: 0;
+    background-image: linear-gradient(to right, #fff1eb 0%, #ace0f9 100%);
+    padding: 15px;
+}
+
+header {
+    display: flex;
+    flex-direction: row;
+}
+
+.titleText {
+    font-weight: 600;
+    line-height: 110%;
+    width: 150px;
+    height: 30px;
+    margin-top: 0;
+    margin-left: 10px;
+}
+
+.subTitle {
+    font-weight: 400;
+}
+
+
+nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    width: calc(100vw - 150px);
+}
+
+nav a {
+    color: inherit;
+    text-decoration: none;
+}
+
+nav a:hover {
+    font-size: 110%;
+}
+
+#abigailOnThunderbird {
+    width: 100vw;
+    height: 100vh;
+    object-fit: cover;
+    position: fixed;
+    z-index: -10;
```

Comment:
why are you doing a -10 z index?  are there many elements you're trying to control overlap with?